.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -0,0 +1,93 @@
+---
+name: Iceberg Bug report 🐞
+description: Problems, bugs and issues with Apache Iceberg
+labels: ["kind:bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "
+        <img src='https://iceberg.apache.org/docs/latest/img/Iceberg-logo.png' align='center'>
+        Thank you for finding the time to report the problem!
+
+        We really appreciate the community's efforts to improve Iceberg."
+  - type: dropdown
+    attributes:
+      label: Apache Iceberg version
+      description: >
+        What Apache Iceberg version are you using?
+      multiple: false
+      options:
+        - "0.13.1 (latest released)"
+        - "0.13.0"
+        - "0.12.1"
+        - "0.12.0"
+        - "0.11.1"
+        - "0.11.0"
+        - "0.10.0"
+        - "0.9.1"
+        - "0.9.0"
+        - "main (development)"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: What happened
+      description: Describe what happened.
+      placeholder: >
+        Please provide the context in which the problem occurred and explain what happened. Feel free to include stacktraces. You can include files by dragging and dropping them here.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: What you think should happen instead
+      description: What do you think went wrong?
+      placeholder: >
+        Please explain why you think the behaviour is erroneous.
+  - type: textarea
+    attributes:
+      label: How to reproduce
+      description: >
+        What should we do to reproduce the problem?
+      placeholder: >
+        Please make sure you provide a reproducible step-by-step case of how to reproduce the problem
+        as minimally and precisely as possible. Keep in mind we do not have access to your cluster.
+        Remember that non-reproducible issues will be closed!
+  - type: dropdown
+    attributes:
+      label: Query engine
+      description: What query engine are you using?
+      multiple: false
+      options:
+        - "Apache Spark"
+        - "Apache Flink"
+        - "Apache Hive"
+        - "Trino"
+        - "Presto"
+        - "Dremio"
+        - "Amazon Athena"
+        - "Amazon EMR"
+        - "0.9.1"
+        - "0.9.0"
```

Comment:
Question: Do we really think we should go back that far?

Reply:
I wouldn't expect people to be still on 0.9.0, but where would be the cutoff point? Maybe only the active releases?

Reply:
We don't even really do backports tbh at this point.

If there's a bug, we release a patch version and expect users to upgrade.

Generally speaking, our dependencies are small enough (or more specifically, well shaded) such that this has been ok.

But having people putthe version they're on is still beneficial.

I'd personally stop at 0.11.0 because anybody on anything earlier than that is.... not something I'm looking to debug. But probably not my call to make.

Reply:
I'll remove everything <0.11.0